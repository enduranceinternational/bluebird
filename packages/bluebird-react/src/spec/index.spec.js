import * as bluebird from '..';

describe('bluebird', () => {
  it('matches snapshot', () => {
    expect(bluebird).toMatchSnapshot();
  });
});
