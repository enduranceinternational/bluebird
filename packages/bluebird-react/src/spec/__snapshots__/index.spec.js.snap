// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bluebird matches snapshot 1`] = `
Object {
  "Accordion": [Function],
  "Alert": [Function],
  "Badge": [Function],
  "Breadcrumb": [Function],
  "BreadcrumbItem": [Function],
  "Button": [Function],
  "ButtonGroup": [Function],
  "ButtonToolbar": [Function],
  "Carousel": [Function],
  "CarouselItem": [Function],
  "Checkbox": [Function],
  "Clearfix": [Function],
  "CloseButton": [Function],
  "Col": [Function],
  "Collapse": [Function],
  "ControlLabel": [Function],
  "Dropdown": [Function],
  "DropdownButton": [Function],
  "Fade": [Function],
  "Form": [Function],
  "FormControl": [Function],
  "FormGroup": [Function],
  "Glyphicon": [Function],
  "Grid": [Function],
  "HelpBlock": [Function],
  "Image": [Function],
  "InputGroup": [Function],
  "Jumbotron": [Function],
  "Label": [Function],
  "ListGroup": [Function],
  "ListGroupItem": [Function],
  "Media": [Function],
  "MenuItem": [Function],
  "Modal": [Function],
  "ModalBody": [Function],
  "ModalDialog": [Function],
  "ModalFooter": [Function],
  "ModalHeader": [Function],
  "ModalTitle": [Function],
  "Nav": [Function],
  "NavDropdown": [Function],
  "NavItem": [Function],
  "Navbar": [Function],
  "NavbarBrand": [Function],
  "Overlay": [Function],
  "OverlayTrigger": [Function],
  "PageHeader": [Function],
  "PageItem": [Function],
  "Pager": [Function],
  "Pagination": [Function],
  "Panel": [Function],
  "PanelGroup": [Function],
  "Popover": [Function],
  "ProgressBar": [Function],
  "Radio": [Function],
  "ResponsiveEmbed": [Function],
  "Row": [Function],
  "SafeAnchor": [Function],
  "SplitButton": [Function],
  "Tab": [Function],
  "TabContainer": [Function],
  "TabContent": [Function],
  "TabPane": [Function],
  "Table": [Function],
  "Tabs": [Function],
  "Thumbnail": [Function],
  "ToggleButton": [Function],
  "ToggleButtonGroup": [Function],
  "Tooltip": [Function],
  "Well": [Function],
  "__esModule": true,
  "utils": Object {
    "ValidComponentChildren": Object {
      "count": [Function],
      "every": [Function],
      "filter": [Function],
      "find": [Function],
      "forEach": [Function],
      "map": [Function],
      "some": [Function],
      "toArray": [Function],
    },
    "__esModule": true,
    "bootstrapUtils": Object {
      "__esModule": true,
      "_curry": [Function],
      "addStyle": [Function],
      "bsClass": [Function],
      "bsSizes": [Function],
      "bsStyles": [Function],
      "getClassSet": [Function],
      "prefix": [Function],
      "splitBsProps": [Function],
      "splitBsPropsAndOmit": [Function],
    },
    "createChainedFunction": [Function],
  },
}
`;
